trailblazer/server/api.py

```diff
@@ -9,8 +10,17 @@
 blueprint = Blueprint("api", __name__, url_prefix="/api/v1")
 
 
+def stringify_timestamps(data: dict) -> dict:
+    """Convert datetime into string before dumping in order to avoid information loss"""
+    for key, val in data.items():
+        if isinstance(val, datetime.datetime):
+            data[key] = str(val)
+    return data
+
+
 @blueprint.before_request
 def before_request():
+    """Authentication that is run before processing requests to the application"""
```

Comment:
Is it possible to run trailblazer without activating authentication? Could be nice for development

Reply:
Yes but then anyone in scilifelab could see it. If we add the ability to reload the status and cancel jobs to the ui this will be dangerous

Reply:
I meant if you can choose to start the application with authentication enabled or disabled. Of course I think that we should run our production instance with authentication enabled

Reply:
Even so, it is possible to scan our website for available endpoints. When interacting with them requires no authentication, it will be possible to first query analyses and then cancel /delete them by Id.